dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -50,38 +50,30 @@ class OpticalFlow : public Operator<Backend> {
   using ComputeBackend = typename detail::backend_to_compute<Backend>::type;
 
  public:
-  explicit OpticalFlow(const OpSpec &spec) :
-          Operator<Backend>(spec),
-          quality_factor_(spec.GetArgument<std::remove_const_t<
-                  decltype(this->quality_factor_)>>(detail::kPresetArgName)),
-          grid_size_(spec.GetArgument<std::remove_const_t<
-                  decltype(this->grid_size_)>>(detail::kOutputFormatArgName)),
-          enable_temporal_hints_(spec.GetArgument<std::remove_const_t<
-                  decltype(this->enable_temporal_hints_)>>(
-                  detail::kEnableTemporalHintsArgName)),
-          enable_external_hints_(spec.GetArgument<std::remove_const_t<
-                  decltype(this->enable_external_hints_)>>(
-                  detail::kEnableExternalHintsArgName)),
-          optical_flow_(std::unique_ptr<optical_flow::OpticalFlowAdapter<ComputeBackend>>(
-                  new optical_flow::OpticalFlowStub<ComputeBackend>(of_params_))),
-          image_type_(spec.GetArgument<decltype(this->image_type_)>(detail::kImageTypeArgName)),
-          device_id_(spec.GetArgument<int>("device_id")) {
+  explicit OpticalFlow(const OpSpec &spec)
+      : Operator<Backend>(spec),
+        quality_factor_(spec.GetArgument<std::remove_const_t<decltype(this->quality_factor_)>>(
+            detail::kPresetArgName)),
```

Comment:
When we're at it... can't we just use normal type instead of this decltype contraption?

Reply:
done